services/brig/test/integration/Main.hs

```diff
@@ -64,8 +67,11 @@ runTests iConf bConf otherArgs = do
     lg <- Logger.new Logger.defSettings
     db <- defInitCassandra casKey casHost casPort lg
     mg <- newManager tlsManagerSettings
-
-    userApi     <- User.tests bConf mg b c ch g =<< mkLocalAWSEnv lg awsOpts mg
+    awsEnv <- AWS.mkEnv lg awsOpts mg
+    
+    -- If we're journaling, make sure we have a clean slate
+    for_ (awsEnv^.AWS.journalQueue) $ SQS.execute (view AWS.amazonkaEnv awsEnv) . SQS.purgeQueue
```

Comment:
this is happening before each test, too, so it should not be strictly necessary here, no?  but i don't mind if it remains there.